.github/workflows/ci.yml

```diff
@@ -32,13 +32,23 @@ jobs:
         with:
           distribution: corretto
           java-version: 11
+          cache: sbt
```

Comment:
[Caching](https://github.com/actions/setup-java#caching-packages-dependencies) should improve build times.